java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -35,6 +37,7 @@
 import org.apache.commons.fileupload.servlet.ServletFileUpload;
 import org.apache.commons.fileupload.util.Streams;
 import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringEscapeUtils;
```

Comment:
I didn't want to add a GWT or Guava dependency. There's already a dependency on org.apache.commons.